galley/pkg/runtime/monitoring/monitoring.go

```diff
@@ -71,6 +83,21 @@ var (
 		"galley/runtime/state/type_instances_total",
 		"The number of type instances per type URL",
 		stats.UnitDimensionless)
+	stateTypeVirtualServicesTotal = stats.Int64("galley/runtime/state/virtual_service_info",
+		"The number of valid virtual service known to galley at a point in time",
+		stats.UnitDimensionless)
+	stateTypeDestinationRulesTotal = stats.Int64("galley/runtime/state/destination_rule_info",
+		"The number of valid valid destination rule known to galley at a point in time",
```

Comment:
nit: remove one `valid` from the description, here and in the following stats.

Reply:
Thanks..Fixed